templates/haskell/package.yaml

```diff
@@ -0,0 +1,139 @@
+# Use this file as a reference when you create package.yaml
+# for your new package.
+# If you have multiple packages, you may wish to extract common
+# definitions into a separate file.
+# Why we need this template:
+# • harmless default extensions
+# • a good set of warnings
+# • metadata fields that are better be set
+# • some tricks (e. g. RTS options)
+name:                myproject
+# version is 0 by default to indicate that nothing is ready
+version:             0
+# adjust/add more authors and maintainers if necessary
+author:              Serokell
```

Comment:
As I understand, the author/maintainer should be people within Serokell, and the copyright holder should be Serokell itself. If that's the case, then we can put a value like `"{AUTHOR NAME}"` so people know to substitute their own name.

Reply:
Usually our packages are developed by a team that sometimes changes, so keeping track of all authors is hard. We have some packages developed by a really small number of people where it's easy to name just a few (1-3) authors, but it's a rare case. So I think `Serokell` is a reasonable default. I will expand my comments.